doc/src/manual/style-guide.md

```diff
@@ -447,3 +447,69 @@ julia> h(1)
 
 Thus, use `Int` literals when possible, with `Rational{Int}` for literal non-integer numbers,
 in order to make it easier to use your code.
+
+## Options for referencing positional arguments by name
+Named positional arguments are not supported in Julia, however, some users may still wish to use a named positional arguments code-style supported by some languages.
+An example of a function, in a different language, supporting named positional arguments and keyword arguments is `f(x,y;z)`, which could then be called by either `f(a,b,z=c)` or with `f(x=a,b=y,z=c)`.  There are several workarounds that can provide an experience similar to named positionals.
+Consider the `mod` method, which a user might call with the variables `a` and `b` in the positional arguments:
+
+```julia
+a = 5; b = 2
+mod( a, b )
+```
+
+One approach is to use inline comment blocks when calling the method:
+```julia
+mod( #=x=# a, #=y=# b )
+```
+
+or with end-of-line comments
+
+```julia
+mod( a, # x
+     b, # y
+    )
+```
+
+Furthermore the use of comments permit you to insert hints describing a positional argument, while not interfering with Julia's multiple dispatch on positional arguments:
+
+```julia
+mod( #=Dividend=# a, #=Divisor=# b )
+mod( a, # Dividend
+     b, # Divisor
+    )
+```
+
+Another option is to define your own wrapper for the method using keyword arguments:
+```julia
+import Base.mod
+function mod(; dividend, divisor )
+    return mod( dividend, divisor )
+end
+mod( dividend = a, divisor = b )
+```
+
+A final option is to (re-)assign variable names that are clear within the context of your code.
+For example, instead of
+
+```julia
+a, b, c = get_unit_sale_data()
+d = mod( a, c )
+```
+
+do
+
+```julia
+units_sold, price_per_unit, units_per_package = get_unit_sale_data()
+leftover_units = mod( units_sold, units_per_package )
+```
+
+or
+
+```julia
+a, b, c = get_unit_sale_data()
+units_sold = a
+price_per_unit = b
+units_per_package = c
+leftover_units = mod( units_sold, units_per_package )
+```
```

Comment:
even if some general form of this advisory were to be added, 50 lines is far far too long.

notice that this entire style guide is only around 500 lines. I bet even the most ardent named-positional-argument supporter would admit that a tip to "name your variables" does not deserve 10% of the real estate of the style guide!